.gitignore

```diff
@@ -7,3 +7,4 @@
 /spec/reports/
 /tmp/
 /vendor/
+/.idea/
```

Comment:
If you don't mind, please keep tool-specific ignores in your global excludes file https://git-scm.com/docs/gitignore

Reply:
Can do.

Reply:
JetBrains `.idea` folder has been removed.